kubernetes/config.go

```diff
@@ -0,0 +1,40 @@
+package kubernetes
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func getClientConfig() (*rest.Config, error) {
+	config, err := rest.InClusterConfig()
```

Comment:
Sounds like the `BuildConfigFromFlags` function already falls back to inCluster config: https://godoc.org/k8s.io/client-go/tools/clientcmd#BuildConfigFromFlags

Reply:
Never mind, this only works, if `kubeconfigPath == ""`.